docs/writing-tests/accessibility-testing.mdx

```diff
@@ -95,17 +95,17 @@ Customize the a11y ruleset at the story level by updating your story to include
 
 {/* prettier-ignore-end */}
 
-#### How to disable a11y tests
+#### How to not run a11y tests automatically
```

Comment:
@valentinpalkovic, we can probably use the following heading for accuracy:
```suggestion
#### Turn off automated a11y tests
```

Also, depending on if we use this or any other text, we'll need to update the following [link](https://storybook.js.org/docs/writing-tests/accessibility-testing#disable-a11y-tests-with-the-test-runner) here in the documentation accordingly to prevent it from breaking.